modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/util/BinaryExpressionConverterUtil.java

```diff
@@ -20,6 +20,10 @@ public static <T> Predicate getExpressionPredicate(
 		Column<?, T> column, BinaryExpression.Operation operation, T value) {
 
 		if (Objects.equals(BinaryExpression.Operation.EQ, operation)) {
+			if (value == null) {
```

Comment:
could you change to `Validator.isNull(value)` ?

Reply:
We can't use that here because we're really only interested in the actual null reference and `Validator.isNull(value)` will return true in other [scenarios](https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/Validator.java#L911)